onyo/lib/commands.py

```diff
@@ -404,15 +406,15 @@ def onyo_mv(inventory: Inventory,
         The path to which the source(s) will be moved, or a single
         source directory will be renamed.
 
-    message: str
+    message: str, optional
         An optional string to overwrite Onyo's default commit message.
 
     Raises
     ------
     ValueError
         If multiple source paths are specified to be renamed.
     """
-    sources = [source] if not isinstance(source, list) else source
+    sources = [source] if not isinstance(source, iter) else list(source)
```

Comment:
I don't think so. `isinstance(source, Iterable)` is `True` for strings and dictionaries for example.